handlers/feedback.go

```diff
@@ -32,7 +32,7 @@ func GetFeedback(rend interfaces.Renderer) http.HandlerFunc {
 	})
 }
 
-func getFeedback(w http.ResponseWriter, req *http.Request, url, errorType, description, name, email, lang string, rend interfaces.Renderer) {
+func getFeedback(w http.ResponseWriter, req *http.Request, url, errorType, description, name, useremail, lang string, rend interfaces.Renderer) {
```

Comment:
Suggestion: In Go variable names should use camelCase; again worth updating in next release

```suggestion
func getFeedback(w http.ResponseWriter, req *http.Request, url, errorType, description, name, userEmail, lang string, rend interfaces.Renderer) {
```